EventVisualisation/View/src/Initializer.cxx

```diff
@@ -132,6 +136,8 @@ void Initializer::setupGeometry()
 
     EVisualisationGroup det = static_cast<EVisualisationGroup>(iDet);
     string detName = gVisualisationGroupName[det];
+    LOG(INFO) << detName;
+
     if (settings.GetValue((detName + ".draw").c_str(), false)) {
       if (detName == "TPC" || detName == "MCH" || detName == "MTR" || detName == "MID" || detName == "MFT" || detName == "AD0" || detName == "FMD") { // don't load MUON+MFT and AD and standard TPC to R-Phi view
```

Comment:
O2 detector names are defined here: https://github.com/AliceO2Group/AliceO2/blob/af0e1bd6731c8a146042f789d77053b583b0b052/DataFormats/Detectors/Common/include/DetectorsCommonDataFormats/DetID.h#L153-L156, there is no anymore MTR, ADO, FMD

Reply:
done